localflavor/md/forms.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django import forms
+
+from localflavor.md.choices import COMPANY_TYPES_CHOICES
+from localflavor.md.validators import MDIDNOFieldValidator, MDLicensePlateValidator
```

Comment:
Just to be consistent with other flavors, you should use relative imports for code in this flavor (e.g. `from .choices import REGION_CHOICES`). You should change this in the other files too.